tests/AnnouncementTest.php

```diff
@@ -13,7 +13,11 @@ class AnnouncementTest extends TestCase
      */
     public function testGetSimpleAnnouncement()
     {
-        $results = AnnouncementUtils::getSimpleAnnouncement();
+        $announcementPathFile = '/data/ssp-announcement.php';
```

Comment:
Why this change?

Reply:
Actually, that's a good question. This test doesn't call the getSimpleAnnouncement function now. Shouldn't it?

Reply:
The problem is that `getSimpleAnnouncement` won't ever fail or throw an exception, instead it returns null.

So this now checks the file that that function uses and sees if it works. I guess I could do this instead:
```
        $announcementPathFile = '/data/ssp-announcement.php';
        if (file_exists($announcementPathFile)) {
            $results = AnnouncementUtils::getSimpleAnnouncement();
            $this->assertNotNull($results);
        }
```

which is actually probably a better idea.

Reply:
I figure we want to have it fail if we try to add an announcement, but it doesn't work; however, if no announcement file exists, we want to allow that through